src/common/utils/utils.js

```diff
@@ -30,6 +30,10 @@ export const extendKyWithTenant = (ky, tenantId) => {
   });
 };
 
+export const isConsortiumEnabled = stripes => {
```

Comment:
Depending on the resolution of the comment for [the line](https://github.com/folio-org/ui-receiving/pull/548/files#diff-90dfc2ed3eac3eb990aa1751e1b867b11fde44ff01fe32c33f521d926bc99744R14) it might be redundant